src/Symfony/Component/PasswordHasher/Tests/Command/UserPasswordHashCommandTest.php

```diff
@@ -292,10 +292,6 @@ public function testThrowsExceptionOnNoConfiguredHashers()
      */
     public function testCompletionSuggestions(array $input, array $expectedSuggestions)
     {
-        if (!class_exists(CommandCompletionTester::class)) {
```

Comment:
This one should logically need https://github.com/symfony/symfony/pull/46059 on 6.1.

Reply:
That change is merged up now.